src/Symfony/Bundle/FrameworkBundle/FrameworkBundle.php

```diff
@@ -74,7 +74,10 @@ public function build(ContainerBuilder $container)
         parent::build($container);
 
         $container->addCompilerPass(new RoutingResolverPass());
-        $container->addCompilerPass(new ProfilerPass());
+        if (class_exists(AddConsoleCommandPass::class)) {
+            $container->addCompilerPass(new AddConsoleCommandPass());
+        }
+        $this->addCompilerPassIfExists($container, ProfilerPass::class);
```

Comment:
Or do I have to link to the old Implementation ? 

or the last chance, make an if/else to import the new one or the old one ?